src/common/ShlinkVersions.js

```diff
@@ -19,7 +19,7 @@ const ShlinkVersions = ({ selectedServer, className, clientVersion = SHLINK_WEB_
 
   return (
     <small className={classNames('text-muted', className)}>
-      Client: <b>{normalizedClientVersion}</b> - Server: <b>{serverVersion}</b>
+      Client: <a href="https://github.com/shlinkio/shlink-web-client/releases" rel="noopener noreferrer"><b>{normalizedClientVersion}</b></a> - Server: <a href="https://github.com/shlinkio/shlink/releases" rel="noopener noreferrer"><b>{serverVersion}</b></a>
```

Comment:
Let me suggest some changes:

* Since the project already depends on `react-external-link`, I would use the `ExternalLink` from it, which already adds the `rel="noopener noreferer"` transparently.
* I would not link to the releases section, but to the specific release that you are clicking on.
* In order to keep links color, they need to have the `text-muted` class, otherwise they will show in blue.

This should cover everything:

```jsx
import { ExternalLink } from 'react-external-link';

// ...

Client: <ExternalLink href={`https://github.com/shlinkio/shlink-web-client/releases/${normalizedClientVersion}`} className="text-muted"><b>{normalizedClientVersion}</b></ExternalLink> - Server: <ExternalLink href={`https://github.com/shlinkio/shlink/releases${serverVersion}`} className="text-muted"><b>{serverVersion}</b></ExternalLink>
```

Reply:
I like ALL of the changes you made beside one.. the direct link to this version.
I think it is much easier to check for new updates if links are:
`https://github.com/shlinkio/shlink-web-client/releases/`
and not
`https://github.com/shlinkio/shlink-web-client/releases/${normalizedClientVersion}`

With `https://github.com/shlinkio/shlink-web-client/releases/${normalizedClientVersion}` you are just checking for the version you already have installed which does not provide any info if there are new versions available.